man/fi_mr.3.md

```diff
@@ -776,6 +777,27 @@ field is determined by the value specified through iface.
 ## hmem_data
 The hmem_data field is reserved for future use and must be null.
 
+## page_size
+Page size allows applications to notify the provider the page size to be
+used for an MR allocation. Typically, providers can select the optimal
+page size. In cases where a VA range has zero pages backing it, the
+provider may not know the optimal page size during registration. Rather
+than always use a potential inferior page size, allow apps to specify
+the page size to be used.
+
+Page size must be power of 2. If the provider supports the page size
```

Comment:
Can drop the power of 2 requirement.  The page size must come from the OS supported page sizes...